toc_config.py

```diff
@@ -129,11 +129,12 @@ def parse_chapter(docname, doc, parent):
                     u'min_group_size': config.get(u'min_group_size', 1),
                     u'max_group_size': config.get(u'max_group_size', 1),
                     u'confirm_the_level': config.get(u'confirm_the_level', False),
+                    u'status': config.get(u'status', u'ready'),
                 }
             allow_assistant_viewing = config.get(u'allow_assistant_viewing', app.config.allow_assistant_viewing)
             allow_assistant_grading = config.get(u'allow_assistant_grading', app.config.allow_assistant_grading)
             exercise.update({
-                u'status': u'unlisted',
+                #u'status': u'unlisted',
```

Comment:
Move the status here, so it applies to external exercises too: `u'status': config.get(u'status', u'unlisted'),`.
Also, use `unlisted` as a default (should be used for chapter exercises and RST generates those).